configs/components/fesom/Subjob_tripy.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```

Comment:
I would suggest you turn on error-exit:

```suggestion
#!/bin/bash -e
```